applications/stepswitch/src/stepswitch_local_extension.erl

```diff
@@ -177,19 +190,46 @@ handle_info(_Info, State) ->
 -spec handle_event(kz_json:object(), state()) -> {'reply', []}.
 handle_event(JObj, #state{request_handler=RequestHandler
                          ,resource_req=Request
+                         ,call_id=CallId
                          }) ->
-    case kapps_util:get_event_type(JObj) of
-        {<<"error">>, _} ->
+    case get_event_type(JObj) of
+        {<<"error">>, _, _} ->
             <<"bridge">> = kz_json:get_value([<<"Request">>, <<"Application-Name">>], JObj),
             lager:debug("channel execution error while waiting for execute extension: ~s"
                        ,[kz_util:to_binary(kz_json:encode(JObj))]),
             gen_listener:cast(RequestHandler, {'local_extension_result', local_extension_error(JObj, Request)});
-        {<<"call_event">>, <<"CHANNEL_DESTROY">>} ->
-            gen_listener:cast(RequestHandler, {'local_extension_result', local_extension_success(Request)});
-        {<<"call_event">>, <<"CHANNEL_EXECUTE_COMPLETE">>} ->
+        {<<"call_event">>, <<"CHANNEL_TRANSFEREE">>, _} ->
+            Transferee = kz_call_event:other_leg_call_id(JObj),
+            gen_listener:cast(RequestHandler, {'replaced', Transferee}),
+            gen_listener:add_binding(RequestHandler, ?CALL_BINDING(Transferee));
+        {<<"call_event">>, <<"CHANNEL_TRANSFEROR">>, _} ->
```

Comment:
@k-anderson code was `copied` from `stespswitch_bridge` as attended xfers from A were not working